Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -68,6 +69,11 @@ class TextureCacheBase
 		// Keep an iterator to the entry in textures_by_hash, so it does not need to be searched when removing the cache entry
 		std::multimap<u64, TCacheEntryBase*>::iterator textures_by_hash_iter;
 
+		// This is used to keep track of both:
+		//   * efb copies used by this partially updated texture
+		//   * partially updated textures which refer to this efb copy
+		std::unordered_set<TCacheEntryBase*> refrences;
```

Comment:
As we cache those entries, this set must be cleaned on every release of an entry. So I doubt this will work fine.

What do you think about an u64 counter of all generated efb copies and just store this number of applied copies here?